packages/api/src/jobs/ai-summarize.ts

```diff
@@ -67,21 +80,18 @@ export const aiSummarize = async (jobData: AISummarizeJobData) => {
     const response = await chain.call({
       input_documents: docs,
     })
-
     if (typeof response.text !== 'string') {
       logger.error(`AI summary did not return text`)
       return
     }
-
     const summary = response.text
     const _ = await authTrx(
       async (t) => {
-        return t.getRepository(AISummary).save({
+        return t.getRepository(AITaskResult).save({
+          request: aiTaskRequest,
           user: { id: jobData.userId },
-          libraryItem: { id: jobData.libraryItemId },
-          title: libraryItem.title,
-          slug: libraryItem.slug,
-          summary: summary,
+          libraryItem: aiTaskRequest?.libraryItem,
```

Comment:
I guess there is no need to put `?` here

Reply:
A library item could be optional, because we might have some tasks that don't require them, maybe something along the lines of "define this word"